universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -99,9 +112,17 @@ export class ULIFrameProvider {
     this.setDashboardVisibility(false);
   }
 
+  private setElementVisibility = (element: HTMLDivElement, isVisible: boolean) => {
+    if (isVisible) {
+      Object.assign(element.style, {display: 'block'});
+    } else {
+      Object.assign(element.style, {display: 'none'});
+    }
+  };
+
   private boundOpenDashboard = this.openDashboard.bind(this);
 
-  initUlButton(element: HTMLButtonElement) {
+  initUlButton(element: HTMLButtonElement, isNewNotifications: boolean) {
```

Comment:
Since you have access to `this.isNewNotifications` you can omit this parameter and just access the field directly